system-addon/content-src/components/MessageCenterAdmin/MessageCenterAdmin.scss

```diff
@@ -0,0 +1,101 @@
+
+.messages-admin {
+  $monospace: 'SF Mono', 'Monaco', 'Inconsolata', 'Fira Mono', 'Droid Sans Mono', 'Source Code Pro', monospace;
+  max-width: 996px;
+  margin: 0 auto;
+  font-size: 14px;
+
+  button {
```

Comment:
Could we reuse some of these styles
https://github.com/mozilla/activity-stream/blob/837cea5f88ceac381a95b5c6bdb0f1e3106d2ad2/system-addon/content-src/styles/_activity-stream.scss#L86-L94